source/administration/command-line-tools.rst

```diff
@@ -894,8 +894,32 @@ mattermost export schedule
           --exportFrom string     Unix timestamp (seconds since epoch, UTC) to export data from.
           --timeoutSeconds string Set how long the export should run for before timing out.
 
+mattermost extract-documents-content 
+-------------------------------------
+
+  Description
+    Extracts document content and stores it in the database for file searches available in Mattermost Cloud, and in Mattermost Self-Managed from v5.35. This database migration is strongly recommended so that `users can search for file attachments <https://docs.mattermost.com/help/getting-started/searching.html#searching-for-files>`__ shared in Mattermost prior to v5.35. Otherwise, search results for files shared in the past may be incomplete. The database migration command extracts and indexes the content of past file attachments.
+    
+    Running this migration adds load to your server. For large deployments or teams that share many large text heavy documents, we recommended you review our `hardware requirements <https://docs.mattermost.com/install/requirements.html#hardware-requirements>`__, and test this feature in a staging environment before enabling file search in production. 
+  
+  Format
+    .. code-block:: none
+    
+      mattermost extract-documents-content 
+
+  Example
+    .. code-block:: none
+    
+      extract-documents-content --from=12345
+  
+  Options
+    .. code-block:: none
+    
+      	 --from    Optional. The timestamp of the earliest file to extract, expressed in seconds, since the unix epoch (Jan 1, 1970). (default 0)
+     	 --to 	   Optional. The timestamp of the latest file to extract, expressed in seconds, since the unix epoch (Jan 1, 1970). (default TBD)
```

Comment:
@jespino - What is the default value for the ``--to`` option for this mattermost extract-documents-content  database migration command?